.gitignore

```diff
@@ -27,3 +27,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+# MACK
+.prettierignore
```

Comment:
I don't think we should exclude the prettierignore, could you actually put it back? What was the reason you were thinking we had to do this?

Reply:
There actually wasn't one in the repo - I put one there because I was getting annoyed at the linter changes. I'm not sure if it even did anything. I can remove it from the .gitignore and make that commit!

Reply:
It should be done!

Reply:
Thanks for adding it!